kymatio/scattering1d/filter_bank.py

```diff
@@ -274,73 +251,78 @@ def compute_xi_max(Q):
     return xi_max
 
 
-def compute_params_filterbank(sigma_min, Q, r_psi=math.sqrt(0.5)):
+def scatnet_generator(J, Q, sigma0, r_psi, **unused_kwargs):
```

Comment:
i dont see why we have to rename this to include `scatnet` 🙄 its the same function, just more of a generator.

Reply:
because it's only one of the possible alternatives. in the future i'd like to set up a way to "import" a filterbank specification from another library (librosa for example) and call the Kymatio scattering factory based on that.

Candidates:
HTK, Slaney, ERB, bark, mel, VQT, etc.